cibyl/sources/zuul/git_api.py

```diff
@@ -0,0 +1,215 @@
+import base64
```

Comment:
It would possibly be convenient to move zuul.d files to a 'zuul.d' folder, that way we do not have both sources mixed up and it is easier to keep track of them. I am saying this because the zuul-ci source also has a couple of files named 'api' and it can be confusing.